canonical/canonical.com
# Refactor career base templates and remove stale templates

## Done

- Refactored careers base templates:
  - Removed `/careers/base_template` and `/careers/base-tabs`
  - Careers pages which require vacancies data should extend `/careers/base`, all others should extend `/base_index`
- Drive by's:
  - Removed stale secondary nav templates
  - Fixed indentation

## QA

- View the site locally in your web browser at: https://canonical-com-1008.demos.haus/careers
- Click through the careers bubble and make sure everything looks/works as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5194
